charts/ai-software-templates/chatbot/templates/service-model-server.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}-model-server
     app.kubernetes.io/name: {{ .Release.Name }}-model-server
   name: {{ .Release.Name }}-model-server
-  namespace: {{ .Release.Namespace }}
```

Comment:
same why did we remove the namespace setting question

Reply:
I'm closing in favor of https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/31#discussion_r1923592267

It will be handled similar to what we decide in this conversation